packages/app/src/cli/services/context/prompts.ts

```diff
@@ -13,8 +13,8 @@ import {
 export async function matchConfirmationPrompt(local: LocalSource, remote: RemoteSource) {
   return renderConfirmationPrompt({
     message: `Match ${local.configuration.name} (local name) with ${remote.title} (name on Shopify Partners, ID: ${remote.id})?`,
-    confirmationMessage: `Yes, that's right`,
-    cancellationMessage: `No, cancel`,
+    confirmationMessage: `Yes, match to existing extension`,
+    cancellationMessage: `No, create as a new extension`,
```

Comment:
I think we're reusing this prompt inside `packages/app/src/cli/services/context/identifiers-functions.ts`

Reply:
I guess for unified deployments this is not a problem, but for old style deployments it is. I'll pass something from the outside to distinguish between the two cases.